### DISCUSSION TITLE
Rename "Big" ARG to "pedigree" ARG?

### DISCUSSION THREAD

**Discussion by jeromekelleher** [tskit-dev/what-is-an-arg-paper#277] [General]
A thought occurred to me this morning. The "big" ARG in which we generate RE and CA events in nonancestral material is similar  (equivalent?) to what we'd get if we tracked all of these events in a pedigree. So, a more intuitive way of describing the two types of (classical) ARGs would be the "pedigree" ARG (in which we track events within pedigree ancestors, who may or may not be genetic ancestors) and the "genetic" ARG (in which we only track events within the genetic ancestors).

Perhaps this is also orthogonal to the other way we want to classify ARGs, i.e., via events or genomes. So we could have a "pedigree genome ARG", in which we track the passage of ancestral material through a subset of the pedigree, including "pass through nodes" (something we are currently doing in the msprime simulation context) or we may have a "pedigree event ARG" where we only track nodes in which an event of some type happened (CA, RE, MIGRATION etc).

(Thinking again though, the thing we're simulating in msprime would be the "pedigree genetic genome ARG" or something - we're only tracking through the bits that are ancestral to the sample.)

This classification falls down a bit then with the types of ARG we're mostly interested in - the "genetic genome ARG" and "genetic event ARG".

Thoughts?

**Comment by hyanwong** [comment]
This is a great point. I like it a lot. I think that we can use "ARG" as shorthand for "genetic ARG", requiring the adjective "pedigree" to be put when we mean the pedigree/big ARG.

It will also help link in to the literature on just how big pedigrees can get, even starting with a few samples.

I can re-work the text with this in mind if we all agree.

I have another thought about pass through nodes, which I'll post in another discussion.

**Comment by JereKoskela** [comment]
I don't think the big ARG quite tracks all the pedigree ancestors, at least in the diploid case. I think the accurate thing to say is that it tracks all genetic ancestors of all genetic ancestors. I don't have a problem with the "pedigree ARG" term if it's useful, but we do have to be careful when defining it to avoid confusion with a full pedigree.

**Reply by hyanwong** [reply]
When does it not track the pedigree ancestors? When there is a direct ancestor (essentially, mother or father) of a genome which does not pass any genetic material to its child?

**Reply by hyanwong** [reply]
I guess this could happen if we are focussing on a small region of genome, but if we are tracking a whole chromosome, I think that the pedigree ARG tracks the (W-F) big ARG* if we are guaranteed that there is at least one recombination per chromosome in each generation.

[*] I'm abusing the terminology here, as the big ARG is not a WF process, but I hope the point remains true.

**Reply by JereKoskela** [reply]
I associate pedigrees with a family tree in which every individual has two parents. Maybe that's not the right picture when tracking a single chromosome (?), but it's what makes me think this could also be misleading if not defined carefully.

**Reply by hyanwong** [reply]
I see what you mean. There is an equivalent "haploid pedigree" in which each haploid genome has 2 parents. I think this is the "pedigree" which is traced by a whole-genome big ARG.

But I take your point about abusing the terminology. Hmm
